docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,304 @@
+  spec:
+    allow:
+    app_labels:
+      'teleport.dev/aws-roles-anywhere-profile-arn': '<Var name="Read Only Profile ARN" />'
+    aws_role_arns:
+      - <Var name="Read Only Role ARN" />
+  ```
+
+1. Create the role with the following command:
+  ```code
+  $ tctl create -f example-read-only-access-role.yaml
+  ```
+
+1. Assign the Role to the users you want to allow access. You can also assign it to an Access List or use it in conjunction with Access Requests.
+
+## Step 4/4. Access AWS resources
+
+Now that you have configured the Integration and set up the Role, users can access AWS resources through Teleport.
+
+### Access AWS Management Console
+
+1. Visit the home page of the Teleport Web UI and click **Resources**.
+   If the Integration found any Profiles as expected, the Web UI will display the name of the Profile.
+
+1. Click the **Launch** button for the AWS Console application, then click on the role you would like to assume when signing in to the AWS Console:
+
+   ![IAM role selector](../../../../img/application-access/iam-role-selector.png)
+
+1. You will get redirected to the AWS Management Console, signed in with the selected role.
+   You should see your Teleport user name as a federated login assigned to `ExampleReadOnlyRole` in the top-right corner of the AWS Console:
+
+   ![Federated login](../../../../img/application-access/federated-login@2x.png)
+
+### Access AWS using AWS CLI or other AWS SDK based tools
+
+#### Obtain the credentials
+On your desktop, log into the AWS App Profile that was synced:
+
+```code
+$ tsh apps login --aws-role arn:aws:iam::278576220453:role/ExampleReadOnlyAccess example-read-only-access
+Logged into AWS app "example-read-only-access".
+
+Your IAM role:
+    arn:aws:iam::123456789012:role/ExampleReadOnlyAccess
+
+Example AWS CLI commands:
+    aws --profile example-read-only-access s3 ls
+    AWS_PROFILE=example-read-only-access aws s3 ls
+```
+
+#### Access AWS using AWS cli
+You can now access all your AWS resources from the command line using the `aws` command line tool.
+
+You need to pass the `--profile` flag or export the `AWS_PROFILE` environment variable with the name of the profile you created earlier.
+
+```code
+$ aws --profile example-read-only-access s3 ls
+...
+```
+
+#### Access AWS from Terraform
+Using Terraform requires you to set the profile in the provider.
+```hcl
+provider "aws" {
+  profile = "example-read-only-access"
+  // ...
+}
+```
+
+Setting the `AWS_PROFILE` environment variable is also an option.
+
+#### Set the default profile
+Instead of always setting the AWS Profile in your commands/scripts/configurations, when logging in, you can set it to be the default AWS configuration profile.
+```code
+$ tsh apps login --aws-role arn:aws:iam::278576220453:role/ExampleReadOnlyAccess example-read-only-access --set-as-default-profile
+```
+
+Now, you will be able to run any AWS cli or AWS SDK based tool (including terraform) without any additional change.
```

Comment:
After this, let's also include "next steps" section as we usually do, with pointers to access controls, access requests and access lists.